src/components/CompareResults/RunValues.tsx

```diff
@@ -59,11 +71,23 @@ function RunValues(props: RunValuesProps) {
       </div>
       <div>
         <div className={styles.values}>
-          {values.map((value, index) => (
+          {firstValues.map((value, index) => (
             <div key={`${index}`} className={styles.value}>
               {formatNumber(value)}
             </div>
           ))}
+          {expanded
+            ? lastValues.map((value, index) => (
+                <div key={`${index}`} className={styles.value}>
+                  {value}
```

Comment:
@esanuandra you forgot to use `formatNumber` here (this wasn't there before, I think this appeared when you rebased). Can you please add it when you have the time?